src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -0,0 +1,116 @@
+use anyhow::Result;
+
+use crate::{HypervisorConfig, VcpuThreadIds};
+use kata_types::capabilities::{Capabilities, CapabilityBits};
+
+const VSOCK_SCHEME: &str = "vsock";
+const VSOCK_AGENT_CID: u32 = 3;
+const VSOCK_AGENT_PORT: u32 = 1024;
+
+unsafe impl Send for QemuInner {}
+unsafe impl Sync for QemuInner {}
+
+pub struct QemuInner {
+    config: HypervisorConfig,
+}
+
+impl QemuInner {
+    pub fn new() -> QemuInner {
+        QemuInner {
+            config: Default::default(),
+        }
+    }
+
+    pub(crate) async fn prepare_vm(&mut self, _id: &str, _netns: Option<String>) -> Result<()> {
+        info!(sl!(), "Preparing QEMU VM");
+        Ok(())
+    }
+
+    pub(crate) async fn start_vm(&mut self, _timeout: i32) -> Result<()> {
+        info!(sl!(), "Starting QEMU VM");
+        Ok(())
```

Comment:
Why not `todo!()` here as well?

Reply:
That's because this version of the qemu driver has nothing to do here, it's fine with nothing being done in `prepare_vm()`.

Reply:
True, `todo!()` would have been cleaner in this commit, however this is fixed by a subsequent commit in this very PR.